nodes/persistence-node/src/services/persistence-service/utils/calculateCIDsToTrackAndUntrack.ts

```diff
@@ -92,7 +92,8 @@ const calculateCIDsToUntrack = (
     //If the IPFS hash is not in any index
     if(!cidIndexesMap[info.ipfsHash]) {
       //Untrack the IPFS hash unless the index for which it was previously logged for is not able to be retrieved
-      if(!info.indexes.some(x => unresponsiveIndexMap[x])) {
+      // and if it's not considered lost
+      if(!info.indexes.some(x => unresponsiveIndexMap[x]) && info.status !== Status.Lost) {
```

Comment:
If the package is lost then we will forever keep a record of it? That doesn't sound right

Reply:
Ok, if we don't need to care about them, then do we even need this part:

https://github.com/polywrap/nodes/pull/128/files#diff-c8ab1fb7d43f3b45c4468d7ecf0ce060d8003c30065893bbcc7734e4b567b2c6L199

Should we just remove them immediately at that point?

Reply:
(and then we maybe don't even need a `Lost` status)

Reply:
We need to keep track of all hashes that the indexers have. If we untracked a hash when it is lost, then the next time we query the indexers (every 15 secs) we would get that hash back and would have to re-process is again (retry until lost), and again..